src/components/app-wrapper.js

```diff
@@ -47,9 +49,6 @@ import { PARAM_THEME } from '../utils/config-params';
 const lightTheme = createTheme({
     palette: {
         mode: 'light',
-        customButton: {
-            main: '#000000DE',
-        },
```

Comment:
hum I don't get it, those custom colors were already there ?

Reply:
Yes, that was the first solution I had implemented

Reply:
ok